front/src/theme.js

```diff
@@ -12,5 +12,6 @@ export default {
   title: "#FFFFFF",
   secondary: "#d3d3d3",
   border: "#ccc",
-  shadow: "rgba(0, 0, 0, 0.17)"
+  shadow: "rgba(0, 0, 0, 0.17)",
+  buttonColor: "#ededed"
```

Comment:
is this the only color we will use for buttons?

Reply:
Yes. Parameterizing it is still a good idea, in case tech united wants it changed in the future.

Reply:
I honestly don't know.